### DISCUSSION TITLE
Dark mode

### DISCUSSION THREAD

**Discussion by dicastropr** [Kittyfisto/Tailviewer#354] [General]
Pretty simple and great tool! Any plans for a Dark mode option?

**Comment by Kittyfisto** [comment]
There were no plans for it just yet, but it sounds like a reasonable requirement.
I've never had a particular problem with bright backgrounds and have only recently adopted to using a dark mode in my IDEs, but it should be simple enough to provide one for Tailviewer.

Feel free to create an enhancement and I'll contact you once this is available.

**Comment by PwrSrg** [comment]
**AGREED**.  Lack of dark mode is the ONLY reason I don't use this.  Otherwise, it would be AMAZING.

**Comment by Quboid** [comment]
I don't know if this is actively being developed but dark mode? Yes please!
